src/backend/marsha/core/static.py

```diff
@@ -1,64 +1,32 @@
 """Customizing Django storage backends to enable blue/green deployments."""
-from collections import OrderedDict
 import re
 
 from django.conf import settings
-from django.contrib.staticfiles.storage import ManifestStaticFilesStorage
 
-from whitenoise.storage import CompressedStaticFilesMixin
-
-
-# from whitenoise.compress import Compressor
+from whitenoise.storage import CompressedManifestStaticFilesStorage
 
 
 STATIC_POSTPROCESS_IGNORE_REGEX = re.compile(
     getattr(settings, "STATIC_POSTPROCESS_IGNORE_REGEX", "^$")
 )
 
-STATIC_POSTPROCESS_MAP_IGNORE_REGEX = re.compile(
-    getattr(settings, "STATIC_POSTPROCESS_MAP_IGNORE_REGEX", "^$")
-)
-
 
-class MarshaCompressedManifestStaticFilesStorage(
-    CompressedStaticFilesMixin, ManifestStaticFilesStorage
-):
+class MarshaCompressedManifestStaticFilesStorage(CompressedManifestStaticFilesStorage):
     """Custom staticfile storage.
 
     Custom staticfile storage allowing us to not post process multiple times
     all the webpack build. We want to ignore it and just compress all js files
     but ignore map files
     """
 
-    # pylint: disable=arguments-differ,unused-argument
-    def post_process(self, paths, dry_run=False, **options):
-        """Remove paths from file to post process.
-
-        All js and map files generated by webpack should be removed from the
-        paths variable to not be post-processed and then only js files are compressed,
-        not map files.
-        """
-        filtered_paths = OrderedDict()
-        to_compress = []
-        for path in paths:
-            if not STATIC_POSTPROCESS_IGNORE_REGEX.match(path):
-                # print("processed  : ", path)
-                filtered_paths[path] = paths[path]
-            elif not STATIC_POSTPROCESS_MAP_IGNORE_REGEX.match(path):
-                # do not compress map files
-                # print("compressed : ", path)
-                to_compress.append((path, None, False))
-            # else:
-            #     print("ignored    : ", path)
-
-        # static files without webpack build are post-processed
-        yield from super().post_process(filtered_paths, dry_run=dry_run, **options)
-        if not dry_run:
-            # static files built from webpack are just compressed
-            yield from super().post_process_with_compression(to_compress)
-
-    # def create_compressor(self, **kwargs):
-    #     """Create compressor instance."""
-    #     kwargs["quiet"] = False
-    #     print("kwargs", kwargs)
-    #     return Compressor(**kwargs)
+    def hashed_name(self, *args, **kwargs):
+        """Exclude files from getting a hashed name."""
+        path = args[0]
+        if STATIC_POSTPROCESS_IGNORE_REGEX.match(path):
+            return path
+        return super().hashed_name(*args, **kwargs)
+
+    def create_compressor(self, **kwargs):
```

Comment:
I left it because when collectstatic runs, nothing is displayed.
With this, we can see what's going on, and maybe debug something.
But I'll remove it if you don't think it's useful.

Reply:
I don't understand why they don't use a logger and you are free then to configure it or not...